mkdocs.yml

```diff
@@ -20,6 +20,5 @@ pages:
         - "Using AoT with Expressive and zend-servicemanager": cookbook/aot-guide.md
     - "Migration Guide": migration.md
 site_name: zend-di
-site_description: zend-di
+site_description: "Automated dependency injection and instance manager."
```

Comment:
There is no instance manager in v3 anymore, in favor of psr containers and ServiceManager.

Reply:
The description was taken from the [component list](https://zendframework.github.io/zf-mkdoc-theme/scripts/zf-component-list.json) which is used for the component overview on [docs.zendframework.com](https://docs.zendframework.com/).

Do you have an idea to renew the description? (The description in the readme file of this repository is too long.)

Reply:
Hmm, how about the following descriptions?

> Automated dependency injection for PSR-11 containers

A less generic one but more ZF specific could be:

> Automated dependency injection for ZendServicemanager

Thoughts? I guess the component list and composer.json description should be updated then as well.

Reply:
> Automated dependency injection for PSR-11 containers

👍 

> I guess the component list and composer.json…

Right! And also the repository description.

Reply:
> And also the repository description.

I can't do that. Can you? Otherwise @Ocramius / @weierophinney may jump in.

Reply:
@weierophinney and @ezimuel